src/features/filter-transactions/TransactionTypeFilter.vue

```diff
@@ -2,32 +2,34 @@
   <BaseTabs
     v-model="model"
     :items="tabs"
+    :adaptive-options="props.adaptiveOptions"
   />
 </template>
 
 <script setup lang="ts">
-import { useI18n } from 'vue-i18n';
 import type * as ftm from './model';
 import { useVModel } from '@vueuse/core';
 import BaseTabs from '@/shared/ui/components/BaseTabs.vue';
+import type { AdaptiveOptions } from '@/shared/types';
 
 interface Props {
-  modelValue: ftm.Tab
+  modelValue: ftm.TransactionTypeTabs | ftm.DefaultTransactionTypeTabs
+  defaultOptions?: boolean
+  adaptiveOptions?: AdaptiveOptions
 }
 
-type Emits = (e: 'update:modelValue', value: ftm.Tab) => void;
+type Emits = (e: 'update:modelValue', value: ftm.TransactionTypeTabs) => void;
 
 const props = defineProps<Props>();
 const emit = defineEmits<Emits>();
-const { t } = useI18n({ useScope: 'global' });
 
 const tabs = [
-  { label: t('all'), value: 'all' },
-  { label: t('transfers'), value: 'transfers' },
-  { label: t('mints'), value: 'mints' },
-  { label: t('burns'), value: 'burns' },
-  { label: t('grants'), value: 'grants' },
-  { label: t('revokes'), value: 'revokes' },
+  { label: props.defaultOptions ? 'transactions' : 'all', value: props.defaultOptions ? 'transactions' : 'all' },
+  { label: 'transfers', value: 'transfers' },
+  { label: 'mints', value: 'mints' },
+  { label: 'burns', value: 'burns' },
+  { label: 'grants', value: 'grants' },
+  { label: 'revokes', value: 'revokes' },
 ];
```

Comment:
This depends on a reactive `props.defaultOptions`. Snouldn't it be `computed`?

Reply:
This computes only once. I think it's enough

Reply:
Then document it! That `defaultOptions` is not supposed to be changed dynamically.